### ISSUE TITLE
FN2 key stuck

### ISSUE THREAD

**Issue by rbirkelbach** [blahlicus/arbites-family#17]
Hello,
it seems that the fn2 key is set to toggle the second layer and does not release it. I currently worked around it by using an additional layer, as this issue does not occur for the fn3 button. This is really frustrating, but I think it should be fairly easy to resolve.

Edit: I was wrong and apparently this behavior occurs for all fn keys I tested. However, it is not observed all the time even with the same key sequences. I am confused.

Edit 2: Now, after a reboot, it works beautifully. This is becoming more and more confusing. 

Best,
Robert

**Comment by blahlicus** [comment]
Hey, I cannot reproduce this on a Diverge 3, the only way to cause this problem is to have toggleFN keys in certain layers but not in certain layers, Arbites automatically assigns FN and toggleFN keys to all layers of the same key so this problem should not occur, would you mind posting the offending keyboard type and layout file for me?

**Comment by rbirkelbach** [comment]
Hi,
it seems to be magically resolved after a reboot. Also, I was not precise, as it was the spacefn keyset, not the fn  keys. The issue was on a diverge tm on Ubuntu 16.04. If it occurs again, I'll get back to you.
Best,
Robert

**Comment by rbirkelbach** [comment]
This behavior reoccurred. Sometimes the Fn2 key gets stuck, sometimes it does not get stuck and sometimes it does not work at all. This is on a diverge tm 1. Attached you find my layout. I zipped it because github does not allow the file extension. I used Arbites Innova 2.3.5000 -- is there maybe a newer version? I only found this version on the website. How do I find out which firmware version is installed on the arduinos?
[Diverge TM Default Layout.arb2dtm.zip](https://github.com/blahlicus/arbites-family/files/1941301/Diverge.TM.Default.Layout.arb2dtm.zip)
